getAllFeatureFlags/main.go

```diff
@@ -45,9 +47,28 @@ func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyRespo
 	db := database.CreateDynamoDB()
 
 	utils.CheckRequestAllowed(db, utils.ConcurrencyDisablingLambda)
-	
-	featureFlagsResponse, err := getAllFeatureFlags(db)
 
+	corsResponse, err := middleware.CORSMiddleware()(request)
+	if err != nil {
+		log.Printf("CORS error: %v", err)
+		return corsResponse, err
+	}
+
+	if corsResponse.StatusCode != http.StatusOK {
+		return corsResponse, nil
+	}
```

Comment:
- same as in https://github.com/Real-Dev-Squad/feature-flag-backend/pull/128#discussion_r1899158914